packages/user/Config/plugin/wit/world.wit

```diff
@@ -9,6 +9,12 @@ interface branding {
     set-network-name: func(name: string) -> result<_, error>;
 }
 
+interface packaging  {
+    use host:common/types.{error};
+    
+    set-account-sources: func(accounts: list<string>) -> result<_, error>;
+}
+
 interface producers {
     use host:common/types.{error};
     use sites:plugin/types.{file};
```

Comment:
You don't use `sites:plugin/types.{file}` in either this interface or the `branding` interface